packages/apps/src/Menu/Grouping.tsx

```diff
@@ -6,90 +6,228 @@ import { Group } from './types';
 
 import React from 'react';
 import styled from 'styled-components';
-import { Icon } from '@polkadot/react-components';
 
 import Item from './Item';
+import { Icon } from '@polkadot/react-components';
+import { Route } from '@polkadot/apps-routing/types';
 
 interface Props extends Group {
   className?: string;
+  variant?: string;
+  activeRoute?: Route;
 }
 
 const SHA_COL = 'rgba(34, 36, 38, 0.12)';
 const SHA_OFF = '5px';
 
-function Grouping ({ className = '', name, routes }: Props): React.ReactElement<Props> {
-  if (routes.length === 1) {
-    return (
-      <Item
-        isToplevel
-        route={routes[0]}
-      />
-    );
-  }
+function Grouping ({ activeRoute, className = '', name, routes, variant }: Props): React.ReactElement<Props> {
+  if (routes && routes.length === 1) {
+    switch (variant) {
+      case 'active-tab': {
+        return (
+          <>{activeRoute &&
+            <MenuActiveItem className='highlight--color'>
+              <Icon icon={activeRoute.icon} />
+              <span>{activeRoute.text}</span>
+            </MenuActiveItem>}
+          </>
+        );
+      }
 
-  return (
-    <li className={className}>
-      <div className='groupHdr highlight--color-contrast'>
-        <span>{name}</span>
-        <Icon icon='caret-down' />
-      </div>
-      <ul className='groupMenu highlight--bg-light'>
-        {routes.map((route): React.ReactNode => (
+      default: {
+        return (
           <Item
-            key={route.name}
-            route={route}
+            isToplevel
+            route={routes[0]}
           />
-        ))}
-      </ul>
-    </li>
-  );
+        );
+      }
+    }
+  }
+
+  switch (variant) {
+    case 'active-tab': {
+      return (
+        <li className={className}>
+          {activeRoute &&
+          <MenuActiveItem className='highlight--color'>
+            <Icon icon={activeRoute.icon} />
+            <span>{activeRoute.text}</span>
+            {routes && routes.length > 1 &&
+              <Icon
+                className='dropdown'
+                icon={'caret-down'}
+              />
+            }
+          </MenuActiveItem>}
+          {routes && routes.length > 1 &&
+            <ul className='groupMenu tab highlight--bg-light'>
+              {routes.map((route): React.ReactNode => {
+                const itemClassName = (activeRoute && activeRoute.text === route.text) ? 'active highlight--color' : '';
+
+                return <Item
+                  className={itemClassName}
+                  key={route.name}
+                  route={route}
+                />;
+              })}
```

Comment:
```suggestion
              {routes.map((route): React.ReactNode => (
                <Item
                  className={(activeRoute && activeRoute.text === route.text) ? 'active highlight--color' : ''}
                  key={route.name}
                  route={route}
                />;
              ))}
```

Additionally that also allows us to get rid of the `return <Item ` (which should be wrapped in ()'s for readability)